ui/webui/resources/cr_components/certificate_manager/brave_overrides/index.ts

```diff
@@ -0,0 +1,6 @@
+// Copyright (c) 2023 The Brave Authors. All rights reserved.
```

Comment:
nit:
```suggestion
// Copyright (c) 2024 The Brave Authors. All rights reserved.
```

Reply:
Fixed in [14403c9](https://github.com/brave/brave-core/pull/26561/commits/14403c9f9a9a64e6679ec7a5b66890f3200ee5ac)